chat/input-bar/src/InputBar/InputBar.tsx

```diff
@@ -340,11 +340,10 @@ export const InputBar = forwardRef<HTMLFormElement, InputBarProps>(
     };
 
     useAutoScroll(highlightedElementRef, menuRef, 12);
-    useBackdropClick(
-      handleBackdropClick,
-      [focusContainerRef, menuRef],
-      isOpen && withTypeAhead,
-    );
+    useBackdropClick(handleBackdropClick, [focusContainerRef, menuRef], {
+      enabled: isOpen && withTypeAhead,
+      allowPropagation: true,
```

Comment:
Setting `allowPropagation` isn't that simple, unfortunately. We need to make sure the previous element receives focus if the backdrop click was not on a clickable element.

This is how it's currently handled in Menu: https://github.com/mongodb/leafygreen-ui/blob/brooke/exp/packages/menu/src/Menu/Menu.tsx#L98-L124

Reply:
E.g. https://github.com/user-attachments/assets/afea74d1-4502-42e6-9f5f-88dfbb1239c1